assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,34 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => param;
 
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
+let add = (param1, param2) => {
+	return param1 + param2;
+};
 
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
+let subtract = (param1, param2) => {
+	return param1 - param2;
+};
 
-// exampleArray = [1,2,3,4];
+exampleArray = [ 1, 2, 3, 4 ];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+const triple = exampleArray.map((num) => {
+	return num * 3;
+});
```

Comment:
Great job refactoring the function using es6 syntax. We will be regularly be using the map array method.